src/messagingsession/DefaultMessagingSession.ts

```diff
@@ -29,6 +30,7 @@ export default class DefaultMessagingSession implements MessagingSession {
     private readonly reconnectController?: ReconnectController,
     private readonly sigV4?: SigV4
   ) {
+    this.isConnecting;
```

Comment:
Is there some assignment missing?

Reply:
ooops, this is me just learning typescript as working.  Defaulted in param above